libs/features/personalization/preview.js

```diff
@@ -254,10 +254,18 @@ function createPreviewPill(manifests) {
           </div>
         </div>
       </div>
-    </div>
-    <div class="dark">
-      <a class="con-button outline button-l" href="${simulateHref.href}" title="Preview above choices">Preview</a>
     </div>`;
+  const darkDiv = document.createElement('div');
+  const button = document.createElement('a');
+  darkDiv.className = 'dark';
+  button.className = 'con-button outline button-l';
+  button.title = 'Preview above choices';
+  button.text = 'Preview';
+  button.href = simulateHref.href;
+  darkDiv.append(button);
+
+  div.append(darkDiv);
```

Comment:
I see `createTag` is already imported in this file. You could use it to trim down the number of lines to create the two elements. You can still treat `href` differently if needed, but `createTag` would still make this a bit more succint.